src/Components/Server/src/Circuits/RemoteRenderer.cs

```diff
@@ -64,6 +64,24 @@ public Task AddComponentAsync(Type componentType, string domElementSelector)
             return RenderRootComponentAsync(componentId);
         }
 
+        /// <summary>
+        /// Associates the <see cref="IComponent"/> with the <see cref="RemoteRenderer"/>,
+        /// causing it to be displayed in the specified DOM element.
+        /// </summary>
+        /// <param name="componentType">The type of the component.</param>
+        /// <param name="parameters">The parameters for the component.</param>
+        /// <param name="domElementSelector">A CSS selector that uniquely identifies a DOM element.</param>
+        public Task AddComponentAsync(Type componentType, ParameterView parameters, string domElementSelector)
```

Comment:
It looks like instead of `domElementSelector` this is really the sequence number now right?

Reply:
Yes, this is a bit odd. Would be good to change the TypeScript-side code too so that `JS.AttachComponent` doesn't talk about `selector` when it means sequence number.